### PR TITLE
fix: modelina cannot be used in website environments

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR fixes that Modelina cannot be used in browsers and adds a react test to ensure it stay that way.

**Related issue(s)**
Fixes https://github.com/asyncapi/modelina/issues/757

### CODE REVIEW

**Review by Samridhi-98 — DISMISSED** [review]

File: examples/README.md [hunk 1]
```diff
@@ -43,4 +43,5 @@ This directory contains a series of self-contained examples that you can use as
 - [java-from-typescript-type-with-options](./java-from-typescript-type-with-options/) - A basic example that shows how to generate a Java model from a TypeScript type input file along with user provided options.
 - [overwrite-naming-formatting](./overwrite-naming-formatting) - A basic example how to overwrite default naming format constraint in this case, overwriting returning a constant case format.
 - [overwrite-default-constraint](./overwrite-default-constraint/) -  A basic example how to overwrite the entire constraint logic and not just a single single part of the default behavior, in this case overwriting the model naming constraint.
+- [integrate with react](./integrate-with-react/) - A basic example that shows how you can integrate Modelina with React.
```
Comment (magicmatatjahu) [inline examples/README.md:46]:
```suggestion
- [integrate with React](./integrate-with-react/) - A basic example that shows how you can integrate Modelina with React.
```

File: docs/integration.md [hunk 2]
```diff
@@ -5,12 +5,25 @@
 <!-- toc -->
 
 - [Integrate Modelina in a website](#integrate-modelina-in-a-website)
+  * [Security NOTICE](#security-notice)
 - [Integrate Modelina in an AsyncAPI generator template](#integrate-modelina-in-an-asyncapi-generator-template)
 
 <!-- tocstop -->
 
 ## Integrate Modelina in a website
```
Comment (magicmatatjahu) [inline docs/integration.md:13]:
```suggestion
## Integrate Modelina in a browser
```

Should be better :)

File: examples/integrate-with-react/src/App.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            type: 'string',
+            format: 'email'
+          }
+        }
+      };
+      const models = await generator.generate(input);
+      const modelsCode = models.map((model) => model.result).join('\n');
+      setModels(modelsCode);
+    })();
+  }, []);
```
Comment (magicmatatjahu) [inline examples/integrate-with-react/src/App.js:26]: TBH, you don't need to evaluate that code inside component context, you can even create given models outside the component and create them before e.g. rendering main app, but it's only comment from me.

File: examples/integrate-with-react/src/App.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+  return (
+    <div className="App" data-testid="App">
+      {models}
```
Comment (magicmatatjahu) [inline examples/integrate-with-react/src/App.js:30]:
```suggestion
      {models || 'Model generation is processed. Please wait...'}
```

**Review by magicmatatjahu — COMMENTED** [review]
Only few suggestions, but they aren't mandatory :)

File: examples/integrate-with-react/src/App.js (hunk 3 above)
Comment (jonaslagoni) [reply examples/integrate-with-react/src/App.js:26]: You are right yea I just wanted to show how everything could be done within a single function to be completely dynamic 🙂

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**